doc/ref_model/chapters/chapter01.md

```diff
@@ -12,6 +12,18 @@
 ## 1.1 Overview
 The Reference Model (RM) specifies a virtualisation technology agnostic (VM-based and container-based) cloud infrastructure abstraction and acts as a "catalogue" of the exposed infrastructure capabilities, resources, and interfaces required by the workloads. This document has been developed by Cloud iNfrastructure Telco Taskforce (CNTT).  For more information about CNTT, its activities and deliverables, see [Cloud iNfrastructure Telco Taskforce](../../common/chapter00.md#2.0).
 
+**Problem Statement:**
+Based on community consultations, including telco operators, technology suppliers, and software developers, there is a realisation that there are significant technical, operational and business challenges to the development and deployment of VNF/CNF network applications related to the lack of a common cloud infrastructure platform. These include but are not limited to the following:
+
+ - Higher development costs due to the need to develop virtualised/containerised network applications on multiple custom platforms for each operator.
+ - Increased complexities due to the need to maintain multiple versions of applications to support each custom environment.
+ - Lack of testing and validation commonalities, leading to inefficiencies and increased time to market. While the operators will still do internal testing, but using an industry driven verification program based on a common cloud infrastructure would provide a head start.
```

Comment:
"provide a head start" to what? the "but" with a "while" is unnecessary and "perform" rather than "do":
```suggestion
 - Lack of testing and validation commonalities, leading to inefficiencies and increased time to market. While the operators will still perform internal testing, the application developers utilising an industry standard verification program on a common cloud infrastructure would lead to efficiencies and faster time to market.
```